src/CMakeLists.txt

```diff
@@ -54,16 +53,17 @@ add_executable(xcompact3d
                     variables.f90
                     visu.f90
                     xcompact3d.f90)
-#add_executable(xcompact3d ${files_xcompact})
-target_include_directories(xcompact3d PRIVATE ${PROJECT_BINARY_DIR}/decomp2d)
+
 target_link_libraries(xcompact3d PRIVATE decomp2d)
+
 if (MPI_FOUND)
   target_link_libraries(xcompact3d PRIVATE MPI::MPI_Fortran)
 endif (MPI_FOUND)
-if(ADIOS2_FOUND)
-  #target_link_libraries(xcompact3d ${ADIOS_LIBRARIES})
-  target_link_libraries(xcompact3d PRIVATE adios2::fortran_mpi)
-endif()
+
+#if(ADIOS2_FOUND)
+#  #target_link_libraries(xcompact3d ${ADIOS_LIBRARIES})
+#  target_link_libraries(xcompact3d PRIVATE adios2::fortran_mpi)
+#endif()
```

Comment:
I think if this is picked up via 2decomp then the same applies to @mathrack's MKL argument?

Reply:
Ok ADIOS2 is now ok